linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -2,20 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use super::{
```

Comment:
I'm not fond of using `super` (except maybe in testing modules) because it introduces variations on how to write imports that Rustfmt cannot normalize.